modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/price/page.jsp

```diff
@@ -7,63 +7,73 @@
 
 <%@ include file="/price/init.jsp" %>
 
-<%
-String containerId = StringPool.BLANK;
+<c:choose>
+	<c:when test="<%= commerceChannelId == 0 %>">
+		<div class="alert alert-info mx-auto">
+			<liferay-ui:message key="this-site-does-not-have-a-channel" />
```

Comment:
Same question as above. My suggestion would simply not to display the component if `commerceChannelId == 0`.